build-logic/src/main/kotlin/org.sonarsource.iac.sonarqube.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
+  id("org.sonarqube")
+}
+
+sonar {
+  properties {
+    property("sonar.projectKey", "org.sonarsource.iac:iac")
+    property("sonar.organization", "sonarsource")
+    property("sonar.exclusions", "**/build/**/*")
+    property("sonar.links.ci", "https://cirrus-ci.com/github/SonarSource/sonar-iac")
+    property("sonar.links.scm", "https://github.com/SonarSource/sonar-iac/sonar-iac-plugin")
```

Comment:
I guess it should be without `sonar-iac-plugin` (even if in Maven it was so) - the link is not valid